content/SCALE/SCALETutorials/Credentials/DirectoryServices/ConfigLDAPSCALE.md

```diff
@@ -5,28 +5,50 @@ weight: 20
 aliases:
 tags:
 - scaleldap
-- scaledirectoryservices
+- scaledirserv
 ---
 
 
+
 TrueNAS has an [Open LDAP](https://www.openldap.org/) client for accessing the information on an LDAP server. An LDAP server provides directory services for finding network resources like users and their associated permissions.
+{{< hint info >}}
+You can have either Active Directory or LDAP configured on SCALE but not both.
+{{< /hint >}}
 
 {{< expand "Does LDAP work with SMB?" "v" >}}
-LDAP authentication for SMB shares is disabled unless you have configured and populated the LDAP directory with Samba attributes.
+LDAP authentication for SMB shares is disabled unless you configured and populated the LDAP directory with Samba attributes.
 The most popular script for performing this task is `smbldap-tools`.
 The LDAP server must support SSL/TLS, and you must import the certificate for the LDAP server CA.
```

Comment:
This is not necessarily true. We just need to be able to validate the full cert chain (e.g. no self-signed certs), if self-signed cert is used, then some hackery like this is needed.